packages/valory/skills/reset_pause_abci/rounds.py

```diff
@@ -54,7 +54,11 @@ def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Event]]:
         """Process the end of the block."""
         if self.threshold_reached:
             extra_kwargs = {}
-            for key in self.synchronized_data.db.cross_period_persisted_keys:
+            cross_period_persisted_keys = (
+                self.synchronized_data.db.cross_period_persisted_keys
+            )
+            cross_period_persisted_keys.update({"consensus_threshold"})
```

Comment:
Why? Isn't the default set of `cross_period_persisted_keys` hard coded elsewhere and should be updated there?

Reply:
Yes, this is set here:

https://github.com/valory-xyz/open-autonomy/blob/ee71f2f577c7baa906508e4003c0b9090a3168b7/packages/valory/skills/abstract_round_abci/base.py#L2065

However, each abci app can overwrite this, e.g., in [price oracle deployment](https://github.com/valory-xyz/price-oracle/blob/f42f53add155a74e89d1bd1e9d47c3e9456daa4c/packages/valory/skills/oracle_deployment_abci/rounds.py#L225-L228).

Therefore I performed this weird change. Also, a reason I opened [[Refactor]: db creation in `ResetAndPauseRound`](https://trello.com/c/nBsvTlEI).